pkg/client/client.go

```diff
@@ -505,6 +505,12 @@ type ImmuClient interface {
 	//
 	// Note: Currently such transaction can only be used for SQL operations.
 	NewTx(ctx context.Context) (Tx, error)
+
+	// TruncateDatabase truncates a database.
+	// This truncates the locally stored value log files used by the database.
+	//
+	// This call requires SysAdmin permission level or admin permission to the database.
+	TruncateDatabase(ctx context.Context, r *schema.TruncateDatabaseRequest) (*schema.TruncateDatabaseResponse, error)
```

Comment:
we should avoid schema structs as much as possible from the client APIs, in this case it would be enough with the context and the retention period as parameters and an error as response, unless more info is returned e.g. up to which transaction ID truncation was done

Reply:
noted, removed it